docs/sources/administration/organization-preferences/index.md

```diff
@@ -237,3 +237,36 @@ You can choose your own personal home dashboard. This setting overrides all home
 1. On the left menu, hover your cursor over your avatar and then click **Preferences**.
 1. In the **Home Dashboard** field, select the dashboard that you want to use for your home dashboard. Options include all starred dashboards.
 1. Click **Save**.
+
+## Change Grafana language
+
+### Change server language
+
+Grafana server administrators can change the default Grafana UI language for all users on the server by setting the [default_language]({{< relref "../../setup-grafana/configure-grafana/#default-language" >}}) option in the Grafana configuration file.
+
+### Change organization language
+
+Organization administrators can change the language for all users in an organization.
+
+1. Hover your cursor over the **Configuration** (gear) icon.
+1. Click **Preferences**.
+1. In the Preferences section, select the **Language**.
+1. Click **Save**.
+
+### Change team language
+
+Organization and team administrators can change the language for all users in a team.
+
+1. Hover your cursor over the **Configuration** (gear) icon in the side menu.
+1. Click **Teams**. Grafana displays the team list.
+1. Click on the team that you want to change the language for and then navigate to the **Settings** tab.
+1. In the Preferences section, select the **Language**.
+1. Click **Save**.
+
+### Change your personal language
+
+You can change the language for your user account. This setting overrides language settings at higher levels.
```

Comment:
I'm not sure what 'higher levels' means. Does this refer to global settings? If so, then maybe "This setting overrides language settings configured globally"?

Reply:
this verbiage is copy-pasted from other preferences on this page.

See the description at the top of the page for an explanation https://github.com/grafana/grafana/blob/d0264b66b85ca64599be07b0e313b166c9a82bb2/docs/sources/administration/organization-preferences/index.md

Reply:
Got it. Ok, this wording is fine with me. Thank you.